scripts/generate-types.ts

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env ts-node
```

Comment:
The main way to run this file should probably be a script in package.json e.g. `"something": "ts-node ./scripts/generate-types.ts"`, so as not to require *nix or a global ts-node install of unknown version